contracts/mocks/MockOracle.sol

```diff
@@ -31,7 +33,15 @@ contract MockOracle {
     }
 
     function getPrice(address _asset) external view returns (uint256) {
-        return realTimePrice[_asset];
+        uint256 price = stablePrice[_asset];
+
+        if (price == 0) {
+            require(assetPricer[_asset] != address(0), "Oracle: Pricer for this asset not set");
+
+            price = OpynPricerInterface(assetPricer[_asset]).getPrice();
+        }
+
+        return price;
```

Comment:
I don't know why do we need to update the mock contract. The mock contract is supposed to be easily return the mocked value, right? Why using calls to external contract instead of just mock the value?

Reply:
Oh yeah, I updated the mock contract to add the stable price feature, cause it didn't exist before, but yeah we can still not call the pricer and return realTimePrice. But I got different errors so ended up having mock similar to the original contract

Reply:
The Compound pricer unit tests use MockOracle file, I think if we don't add the pricer in the MockOracle, then the case of cETH in compound pricer tests does not make sense as we are not using the underlying pricer for testing (WETH pricer)